extended/jams2015.tex

```diff
@@ -217,7 +218,7 @@ \subsection{Task- vs. Annotation-major layout}\label{sec:schema:task}
 
 As illustrated in \cref{jams1}, JAMS-0.1 took a \emph{task-major} approach to structuring annotations.  A collection of supported tasks was
 defined within the JAMS-0.1 schema, such as \emph{tag}, \emph{genre}, \emph{chord}, \emph{key}, \emph{melody}, \emph{etc}.  Annotations of a
-particular task would then be accessed by indexing the array of annotations corresponding to that task, \emph{e.g.}:
+particular task would then be accessed by indexing the array of annotations corresponding to that task, \emph{e.g.}:\\
```

Comment:
the next object's top border was getting cut off. this was a hacky way to get it back.